lib/server/ServerRequest.js

```diff
@@ -3,11 +3,13 @@
 var ServerRequest = {
 
     from: function(expressRequest, environmentConfig) {
+        var rawQuery = (expressRequest.originalUrl && expressRequest.originalUrl.indexOf('?') != -1) ? expressRequest.originalUrl.substring(expressRequest.originalUrl.indexOf('?') + 1) : null;
```

Comment:
We have qs as a dependency, could that help you here? It would also be nice if this was its own function.